Tests/SentryTests/SentryKSCrashReportConverterTests.m

```diff
@@ -235,6 +235,15 @@ - (void)testStackoverflow
 - (void)testCPPException
 {
     [self isValidReport:@"Resources/CPPException"];
+
+    NSDictionary *rawCrash = [self getCrashReport:@"Resources/CPPException"];
+    SentryCrashReportConverter *reportConverter =
+        [[SentryCrashReportConverter alloc] initWithReport:rawCrash inAppLogic:self.inAppLogic];
+    SentryEvent *event = [reportConverter convertReportToEvent];
+
+    SentryException *exception = event.exceptions.firstObject;
+
+    XCTAssertEqualObjects(exception.value, @"MyException: Something bad happened...");
```

Comment:
`h`: I did not see the test failing when changing the `type` of the SentryException.

Reply:
Good catch thanks 🥇

Reply:
Good catch thanks 🥇